examples/guards/lib/main.dart

```diff
@@ -224,11 +225,11 @@ class MyApp extends StatelessWidget {
       ],
     ),
     guards: [
-      // Redirect to /login if the user is not authenticated:
+      // Guard /books/* by beaming to /login if the user is unauthenticated:
       BeamGuard(
-        pathBlueprints: ['/books*'],
+        pathBlueprints: ['/books/*'],
```

Comment:
I changed this from `/books*` to `/books/*`  because it could happen in complex apps that when another route also starts with `/books` (e.g. `/books_new`) this is also guarded, but maybe shouldn't.

Unfortunately, I noticed that with this change  `/books` is not guarded any more.  Is that intentional?

Reply:
Oh, I see. We can then do `['/books','/books/*']`. I'll tweak that soon